build/images/test/start_ovs_netdev

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source logging
+source daemon_status
+
+CONTAINER_NAME="antrea-ovs"
+OVS_RUN_DIR="/var/run/openvswitch"
+OVS_DB_FILE="/var/run/openvswitch/conf.db"
+
+if ! ip addr show eth0 > /dev/null 2>&1; then
+    log_error $CONTAINER_NAME "No eth0 uplink found, exiting"
+    exit 1
+fi
+
+# While working on https://github.com/antrea-io/antrea/pull/2215, we sometimes
+# observed a few leftover .ctl files across OVS restarts, causing failures in
+# the agent when trying to run ovs-appctl commands. This impacted testing on
+# Kind, so we started deleting these files before starting OVS.
+function cleanup_ovs_run_files {
+    rm -rf ${OVS_RUN_DIR}/ovs*.pid
+    rm -rf ${OVS_RUN_DIR}/ovs*.ctl
+    rm -rf ${OVS_RUN_DIR}/.conf.db.*~lock~
+}
+
+function start_ovs {
+    cleanup_ovs_run_files
+    log_info $CONTAINER_NAME "Starting OVS"
+    # Unlike the start_ovs script, we don't set flow-restore-wait when starting OVS
+    # with the netdev datapath. This is because the Node's network relies on the
+    # forwarding of OVS so we cannot get Node, Pod, NetworkPolicy data from
+    # Kubernetes API to install proper flows before removing flow-restore-wait.
+    /usr/share/openvswitch/scripts/ovs-ctl --system-id=random start --db-file=$OVS_DB_FILE
+}
+
+function stop_ovs {
+    log_info $CONTAINER_NAME "Stopping OVS"
+    /usr/share/openvswitch/scripts/ovs-ctl stop
+}
+
+SLEEP_PID=
+
+function quit {
+    log_info $CONTAINER_NAME "Stopping OVS before quit"
+    stop_ovs
+    # terminate background sleep process
+    if [ "$SLEEP_PID" != "" ]; then kill $SLEEP_PID > /dev/null 2>&1 || true; fi
+    exit 0
+}
+
+set -euo pipefail
+
+# Do not trap EXIT as it would then ignore the "exit 0" statement in quit and
+# exit with code 128 + SIGNAL
+trap "quit" INT TERM
+
+start_ovs
+
+if [[ "$#" -ge 1 ]] && [[ "$1" == "--start-ovs-only" ]]; then
+  exit 0
+fi
```

Comment:
all of this could be replaced with just:
```bash
set -euo pipefail
start_ovs
```

In integration tests, the script is only used to start OVS.
We don't need to keep the `--start-ovs-only` flag.
The "stop" logic is never used since the script returns right away.

Reply:
You are right, I have simplified the script as you said.